[2024-07-08T21:54:39.904+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-08T21:54:39.972+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.get_files scheduled__2024-07-07T00:00:00+00:00 [queued]>
[2024-07-08T21:54:39.979+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.get_files scheduled__2024-07-07T00:00:00+00:00 [queued]>
[2024-07-08T21:54:39.980+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-08T21:54:40.002+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_files> on 2024-07-07 00:00:00+00:00
[2024-07-08T21:54:40.020+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'get_files', 'scheduled__2024-07-07T00:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp9_qd3n1c']
[2024-07-08T21:54:40.023+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=13743) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-08T21:54:40.024+0000] {standard_task_runner.py:63} INFO - Started process 13768 to run task
[2024-07-08T21:54:40.025+0000] {standard_task_runner.py:91} INFO - Job 2: Subtask get_files
[2024-07-08T21:54:40.087+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.get_files scheduled__2024-07-07T00:00:00+00:00 [running]> on host 8c15d98f81a6
[2024-07-08T21:54:40.211+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-07-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-07T00:00:00+00:00'
[2024-07-08T21:54:40.213+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-08T21:54:40.214+0000] {logging_mixin.py:188} INFO - 2 files found in /opt/***/dags/data
[2024-07-08T21:54:40.214+0000] {python.py:237} INFO - Done. Returned value was: ['/opt/***/dags/data/github_events_01.json', '/opt/***/dags/data/github_events_02.json']
[2024-07-08T21:54:40.214+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-08T21:54:40.243+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl, task_id=get_files, run_id=scheduled__2024-07-07T00:00:00+00:00, execution_date=20240707T000000, start_date=20240708T215439, end_date=20240708T215440
[2024-07-08T21:54:40.280+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-08T21:54:40.329+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-08T21:54:40.354+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
